pages/cart.vue

```diff
@@ -144,28 +162,54 @@ export default defineComponent({
       router.push({ path: "/checkout" })
     }
 
+    const applyPromocode = async (couponApplied) => {
+      await applyCoupon(couponApplied)
+    }
+
+    const isValidCoupon = computed(() => {
+      return !cart.value?.invalidCoupons[0]?.couponCode
+    })
+
+    const invalidCouponErrorText = computed(() => {
+      return `${cart.value?.invalidCoupons[0]?.couponCode} is an invalid code`
+    })
+
+    const appliedCoupons = computed(() => {
+      return cart.value?.couponCodes
+    })
+
+    const areCouponsApplied = computed(() => {
+      return cart.value?.couponCodes.length
+    })
+
+    const productAppliedCoupons = (cartItem) => {
+      return cartGetters.getProductAppliedCoupons(cartItem)
+    }
```

Comment:
Can we simplify it like below (for all above the places)?
const productAppliedCoupons = (cartItem) => cartGetters.getProductAppliedCoupons(cartItem)

Reply:
Done